RELEASE-NOTES.md

```diff
@@ -1,5 +1,73 @@
 # Prefect Release Notes
 
+## Release 2.9.0
+
+### Track and manage artifacts
+
+Most workflows produce or update an artifact of some kind, whether its a table, a file, or a model. With Prefect Artifacts, you can track changes to these outputs and richly display them in the UI as tables, markdown, and links. Artifacts may be associated with a particular task run, flow run, or even exist outside a flow run context, enabling you to not only observe your flows, but the objects that they interact with as well. 
+
+<insert image>
+
+### Result Storage Keys
+
+While Prefect allowed you to persist results to a storage block, the path could only be a UUID, so you were limited in how those results could be organized. This release introduces result storage keys, which enable you to configure the path of the result file in the result storage. Result storage keys can be formatted with access to all of the modules in `prefect.runtime` and the run's `parameters`. For example, you can name each result to correspond to the flow that produced it:
```

Comment:
I think I'd say something like..

```suggestion
When persisting results, Prefect stores data at a unique randomly generated path. While this is convenient for ensuring the result is never overwritten, it limits organization of result files. In this release, we've added configuration of result storage keys, which gives you control over the result file path. Result storage keys can be dynamically formatted with access to all of the modules in `prefect.runtime` and the run's `parameters`. 
```

Worth noting this doesn't actually template anything from the parent flow in the below example so the previous example intro didn't make sense. We can make it a new paragraph.